next.config.js

```diff
@@ -1,8 +1,8 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  experimental: {
-    appDir: true,
-  },
+  // experimental: {
+  //   appDir: true,
+  // },
```

Comment:
if its not needed please remove instead of comment 👍🏻 

in general one PR should only address one thing, but in this case we can do an exception because its very small

Reply:
Ok, I am doing that now